examples/templates/kubernetes/main.tf

```diff
@@ -262,8 +262,9 @@ resource "kubernetes_deployment" "main" {
       }
       spec {
         security_context {
-          run_as_user = 1000
-          fs_group    = 1000
+          run_as_user     = 1000
+          fs_group        = 1000
+          run_as_non_root = True
```

Comment:
@Cjkjvfnby - thank you for your contribution. this will need to be lowercase `true` to pass K8s validation checks.